czishrink/libczicompressc/libczicompressc.nuspec

```diff
@@ -11,6 +11,6 @@
     <license type="expression">MIT</license>
     <projectUrl>https://github.com/zeissmicroscopy/czicompress/</projectUrl>
     <copyright>© Carl Zeiss Microscopy GmbH and others. All rights reserved.</copyright>
-    <repository type="git" url="https://github.com/ZEISS/czicompress" branch="main" commit="029973e80f064bce28b22122213e327037eef782" />
+    <repository type="git" url="https://github.com/ZEISS/czicompress" branch="main" commit="40a2ba8489544626c2481ca95177b3235b5c0de7" />
```

Comment:
Should this now point towards the latest CziCompress CAPI change or the latest commit of the repo? If the latter we may want to automate settings this via PS script

Reply:
It is already automated: See https://github.com/ZEISS/czicompress/blob/b2c6171fe8500a3015331a7ff7f9847bb27d5168/czishrink/upgrade-libczicompressc.ps1#L248

Reply:
already done in script